eventActions/reactions.js

```diff
@@ -60,17 +60,17 @@ const handleReactions = async function(client, message, isCommand) {
 			Config.CHANNELS.CITADEL,
 			Config.EMOTES.GOOD_MORNING,
 			Config.STRICT_GREETINGS
-				? new RegExp('mo*rning (koa[^a-z]*|knights[^a-z]*|friends[^a-z]*|everyone[^a-z]*)$')
-				: new RegExp('g+o{2,}d+\\s*m+o+r+n+i+n+g+')
+				? new RegExp('mo*rning (koa|knights|friends|everyone)\\W*$', 'i')
```

Comment:
I'm not 100% sure, but I think `\\W` matches "\W" literally. Try just using `\W`?